### ISSUE TITLE
Not translated

### ISSUE THREAD

**Issue by AlemSpika** [xchacha20-poly1305/husi#16]
![Удалить нерабочие](https://github.com/xchacha20-poly1305/husi/assets/90942166/84d3e9ac-0e85-4559-9a1a-1088b049cd6f)

**Comment by Otnak** [comment]
Translated this line into the menu as "Удалить недоступные".

**Comment by Otnak** [comment]
> If you want to translate, please go to weblate.

Yes, I translated this line to Webplate. As I understand it, it will already be translated in the next build?
